Makefile

```diff
@@ -1,5 +1,17 @@
-test: deps
-	go test -v -coverprofile=cover.out .
+# envVars defines all the environment variables the project depends on. These
+# are typically provided by docker-compose files or .env files.
```

Comment:
I don't get why they're not provided by a `.env` file here?

Reply:
Because these are only used for running the tests and creating a new env file + loading data from it is much more complicated than defining the values right next to the spot where they are used.